spec/lib/whitehall_importer/integrity_checker/change_history_check_spec.rb

```diff
@@ -0,0 +1,138 @@
+RSpec.describe WhitehallImporter::IntegrityChecker::ChangeHistoryCheck do
+  let(:published_edition) do
+    build(:edition,
+          document_type_id: "news_story",
+          state: "published")
+  end
+  let(:draft_edition) { build(:edition, document_type_id: "news_story") }
+  let(:public_timestamp) { Time.zone.now.yesterday }
```

Comment:
I imagine we can remove the need for this let - it seems a bit confusingly named as it's not a special timestamp and we can maybe replace it with `Date.yesterday.noon` in each of the tests at a similar number of characters.